src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -252,7 +222,7 @@ export const ArtworkContainer = createFragmentContainer(Artwork, {
 
       ...PartnerCard_artwork
       ...AboutWork_artwork
-      ...OtherWorks_artwork
+      ...OtherWorks_artwork @relay(mask: false)
```

Comment:
cc @ds300 This seems to be the root of the error I mentioned in slack.

Reply:
we resolved this offline by adding both masked and unmasked fragment spreads for OtherWorks!